app/models/cart.rb

```diff
@@ -1,5 +1,6 @@
 class Cart < ApplicationRecord
   has_many :line_items, dependent: :destroy
+  has_many :products, -> { where 'products.enabled = true' }, through: :line_items
```

Comment:
we need to add 2 associations - 
- ​Create​ ​a​ ​association​ ​on​ ​cart​ ​through​ ​which​ ​we​ ​could​ ​get​ ​all​ ​the​ ​products​ ​associated​ ​with the​ ​cart.
- Create​ ​a​ ​association​ ​on​ ​cart​ ​model​ ​through​ ​which​ ​i​ ​could​ ​get​ ​all​ ​enabled​ ​products associated​ ​with​ ​it.

Are we doing it here?

Reply:
done